ui/src/editor/index.ts

```diff
@@ -0,0 +1,160 @@
+import {
+  Editor,
+  mergeAttributes,
+  Node,
+  ToolbarItem,
+  ToolboxItem,
+} from "@halo-dev/richtext-editor";
+import { IconMotionLine } from "@halo-dev/components";
+import {
+  type Component,
+  type FunctionalComponent,
+  markRaw,
+  type SVGAttributes,
+} from "vue";
+import { StickerPluginKey, StickerPmPlugin } from "@/plugin";
+
+// 工具栏接口定义
+export interface ToolbarItem {
+  priority: number;
+  component: Component;
+  props: {
+    editor: Editor;
+    isActive: boolean;
+    disabled?: boolean;
+    icon?: Component;
+    title?: string;
+    action?: () => void;
+  };
+  children?: ToolbarItem[];
+}
+
+export interface ToolboxItem {
+  priority: number;
+  component: Component;
+  props: {
+    editor: Editor;
+    icon?: Component;
+    title?: string;
+    description?: string;
+    action?: () => void;
+  };
+}
+
+export interface CommandMenuItem {
+  priority: number;
+  icon: FunctionalComponent<SVGAttributes>;
+  title: string;
+  keywords: string[];
+  command: ({ editor, range }: { editor: Editor; range: Range }) => void;
+}
+
+export interface StickerOptions {
+  inline: boolean;
+  HTMLAttributes: Record<string, any>;
+  getToolbarItems: ({ editor }: { editor: Editor }) => ToolbarItem;
+  getToolboxItems: ({ editor }: { editor: Editor }) => ToolboxItem;
+  getCommandMenuItems: () => CommandMenuItem;
+}
```

Comment:
Halo 本身导出了 `ExtensionOptions` 接口，这里可以直接使用，不用再重新定义。

```suggestion
export interface StickerOptions {
  inline: boolean;
  HTMLAttributes: Record<string, any>;
}
```